runbotics/runbotics-desktop/src/actions/variables/VariablesActionHandler.ts

```diff
@@ -64,6 +85,8 @@ export class VariablesActionHandler extends StatelessActionHandler {
         let output: any = {};
         switch (action.script) {
             case 'variables.assign':
+                output = await this.assignVariables(action.input, action);
+                break;
             case 'variables.assignList':
                 output = await this.assignVariable(action.input, action);
```

Comment:
change the name of this method to `assignListVariable`, so it will be easier to distinct implementations